aip-console-tools-core/src/main/java/com/castsoftware/aip/console/tools/core/utils/Constants.java

```diff
@@ -21,6 +21,8 @@ private Constants() {
     public static final int RETURN_APPLICATION_NO_VERSION = 9;
     public static final int RETURN_APPLICATION_VERSION_NOT_FOUND = 10;
     public static final int RETURN_INPLACE_MODE_ERROR = 11;
+    // Keep ordinal to 9 fro backward compatibility
+    public static final int RETURN_VERSION_WITH_ANALYSIS_DONE_NOT_FOUND = 9;
```

Comment:
I don't get this ?
Can you explain a bit more what its about, since there already is a static int with value 9 used above ?
Its a static int, not an Enum with an ordinal value, so I'm a bit confused.

Reply:
The returned value was hardcode to 9
So I added constant not to provide different value

Reply:
initial comment and value was
                // FIXME: Constant for return code
                return 9;